### ISSUE TITLE
Fix CI

### ISSUE THREAD

**Issue by ErikSchierboom** [exercism/z3#91]
This repository's `main` branch currently does not pass all CI checks.
This makes detecting issues in PRs harder and might result in a broken track, and so is an important thing to fix.

The following CI workflows are currently failing:

https://github.com/exercism/z3/runs/3372650139
https://github.com/exercism/z3/runs/3372650038
https://github.com/exercism/z3/runs/3372649989
https://github.com/exercism/z3/runs/3372649937
https://github.com/exercism/z3/runs/3372649825

## Tracking

https://github.com/exercism/v3-launch/issues/55

**Comment by theomelo** [comment]
Hello @ErikSchierboom, I'm debugging the workflow locally with [act](https://github.com/nektos/act) - because GH logs for these actions are no longer available - and I'm getting the following error:
```shell
can't open file 'scripts/mk_make.py': [Errno 2] No such file or directory
```
Where can I find `mk_make.py`?

**Comment by ErikSchierboom** [comment]
I have no idea, I wasn't involved in setting up this track, sorry.

**Comment by theomelo** [comment]
Should I ask about it in the forum, then? Given that we're not going to use Slack anymore? I just wonder how I could help fix this error if I can't reproduce it.

**Comment by ErikSchierboom** [comment]
@theomelo Yeah that could work well.

**Comment by ErikSchierboom** [comment]
@theomelo I was wondering, is Z3 actually a language or actually a tool? Looking at the docs, it sounds like it is the latter.

**Comment by theomelo** [comment]
Yes, I see that they call Z3 an ["engine"](https://www.microsoft.com/en-us/research/project/z3-3/), so it's a tool for me. I also found where the `scripts` folder it's referencing to are: https://github.com/Z3Prover/z3/tree/master/scripts

**Comment by theomelo** [comment]
I'll fix the CI script to use the Z3 repo.

**Comment by ErikSchierboom** [comment]
Okay, maybe wait with doing some work on this repo then, as it doesn't really fit our language tracks mold.

**Comment by kytrinyx** [comment]
@theomelo I'm so sorry about the confusion. I didn't do proper due diligence when this track was proposed, and I've now suggested shutting it down: https://github.com/exercism/request-new-language-track/issues/93#issuecomment-1311385254
